keps/prod-readiness/sig-scheduling/3633.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 3633
 alpha:
   approver: "@wojtek-t"
+beta:
+  approver: "@wojtek-t"
```

Comment:
@wojtek-t Hi, I just assigned it to you, but please give it to another person if needed.

Reply:
ACK - please have it reviewed by the SIG first.